irohad/ametsuchi/peer_query.hpp

```diff
@@ -21,25 +21,31 @@
 #include <memory>
 #include <nonstd/optional.hpp>
 #include <vector>
+#include "utils/polymorphic_wrapper.hpp"
 
-namespace iroha {
-
-  namespace model {
-    struct Peer;
-  }
+namespace shared_model {
+  namespace interface {
+    class Peer;
```

Comment:
What is the purpose of `forward declaration` here? And don't use it for polymorphic wrapper?

Reply:
I declare a shared_ptr to Peer, thus I don't need to include the whole file